roles/acme/tasks/main.yml

```diff
@@ -12,21 +12,26 @@
 - name: Run key generation
   ansible.builtin.include_tasks:
     file: create-keys.yml
+  when: acme_domain | length > 0
```

Comment:
are these checks redundant by the precondition, or are there cases left, where the precondition allows this to happen?

Reply:
You're right. I changed this with https://github.com/telekom-mms/ansible-collection-acme/commit/541861ecf069d8da23ee3ca4f1d239ae296f3d8c